libs/design-system/package.json

```diff
@@ -16,6 +16,7 @@
     "dist/types"
   ],
   "scripts": {
+    "postinstall": "panda codegen",
```

Comment:
`panda codegen` is what was recommended in [the docs](https://panda-css.com/docs/installation/cli#update-packagejson-scripts), but I saw somewhere else that someone else was using `postinstall` which helped me avoid an issue when doing a fresh / clean install and build

Reply:
the `postinstall` is running during the `install` lifecycle, but not during the `build` lifecycle, that's why they mention in [the docs](https://panda-css.com/docs/installation/cli) to put it under the `prepare` which is running during both `install` and `build`...

Reply:
also from what I see the `codegen` command only generates the JS (CSS) utilities without the `styles.css` file

Reply:
Correct, it only generates the files without styling, which are required for any build steps since projects refer to the generated code. I'll change back to `prepare` for now